appeals/api/src/server/endpoints/integrations/integrations.controller.js

```diff
@@ -38,9 +54,21 @@ export const postAppealSubmission = async (req, res) => {
 export const postLpaqSubmission = async (req, res) => {
 	const { caseReference, questionnaire, documents } = mapQuestionnaireSubmission(req.body);
 	const result = await createOrUpdateLpaQuestionnaire(caseReference, questionnaire, documents);
-	const formattedResult = mapAppeal(result);
+	if (!result) {
+		throw new BackOfficeAppError(
+			`Failure importing LPA questionnaire. Appeal with case reference '${caseReference}' does not exist.`
+		);
```

Comment:
Ideally this string should be defined in the constants file (appeals/api/src/server/endpoints/constants.js)